pkg/cmd/repo/clone/clone.go

```diff
@@ -131,6 +142,11 @@ func cloneRun(opts *CloneOptions) error {
 	}
 	canonicalCloneURL := ghrepo.FormatRemoteURL(canonicalRepo, protocol)
 
+	// If RepoHasWiki is true and repo opts has wiki then create a new clone URL
+	if canonicalRepo.RepoHasWiki() && wikipref {
+		canonicalCloneURL = strings.Replace(canonicalCloneURL, ".git", ".wiki.git", 1)
```

Comment:
You do not want to accidentally replace a `.git` string that appears in the middle of the URL. It can be only at the end. For that reason, `Replace` is not suitable here. Instead:

```go
canonicalCloneURL = strings.TrimSuffix(canonicalCloneURL, ".git") + ".wiki.git"
```